src/resources/PowerShellWorkerStrings.resx

```diff
@@ -352,7 +352,31 @@
   <data name="DependencySnapshotDoesNotContainAcceptableModuleVersions" xml:space="preserve">
     <value>Dependency snapshot '{0}' does not contain acceptable module versions.</value>
   </data>
+  <data name="FoundExternalDurableSdkInSession" xml:space="preserve">
+    <value>Found external Durable Functions SDK in session: Name='{0}', Version='{1}', Path='{2}'.</value>
+  </data>
+  <data name="IsOrchestrationFailureKey" xml:space="preserve">
+    <value>IsOrchestrationFailure</value>
+  </data>
+  <data name="UnableToInitializeOrchestrator" xml:space="preserve">
+    <value>Unable to initialize orchestrator function due to presence of other bindings. Total number of bindings found is '{0}'. Orchestrator Functions should never use any input or output bindings other than the orchestration trigger itself. See: https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-bindings</value>
```

Comment:
Would it be possible to make this link `https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-bindings` an aka link like `https://aka.ms/functions-powershell-managed-dependency`?

Reply:
Yes, I just created `aka.ms/df-bindings`